client/blocks/importer/wordpress/upgrade-plan/skeleton/index.tsx

```diff
@@ -1,19 +1,96 @@
+import type { FC } from 'react';
 import './style.scss';
 
-interface SkeletonProps {
-	width?: number | string;
-	height?: number | string;
-}
-
-// Borrowed from site-profiler.
-export const Skeleton = ( { width = '100%', height = 20 }: SkeletonProps ) => {
+export const Skeleton: FC = () => {
 	return (
-		<div
-			className="import-upgrade-plan-skeleton"
-			style={ {
-				width,
-				height,
-			} }
-		/>
+		<div className="import__upgrade-plan-details import__upgrade-plan-details--loading">
+			<div className="import__upgrade-plan-period-switcher">
+				<div
+					className="import-upgrade-plan-skeleton"
+					style={ { width: '250px', height: '38px' } }
```

Comment:
Since the spacings and sizes are very dynamic to simulate a real content here, I just created the styles with default classes, and other styles are defined directly in the component.